<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>szukajNaprawy</class>
 <widget class="QDialog" name="szukajNaprawy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="szukajButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wrocButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Wróć</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="numerRejLE"/>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="odDataCB">
      <property name="text">
       <string>Od daty wykonania</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="odKosztCB">
      <property name="text">
       <string>Koszt od</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="doKosztDoubleSpinBox">
      <property name="prefix">
       <string notr="true"/>
      </property>
      <property name="suffix">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <double>999999.989999999990687</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QDoubleSpinBox" name="odKosztDoubleSpinBox">
      <property name="prefix">
       <string notr="true"/>
      </property>
      <property name="suffix">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <double>999999.989999999990687</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="doKosztCB">
      <property name="text">
       <string>Koszt do</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="doDataCB">
      <property name="text">
       <string>Do daty wykonania</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="numerFakCB">
      <property name="text">
       <string>Numer faktury</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="numerFakLE"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="numerRejCB">
      <property name="text">
       <string>Numer rejestracyjny</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QDateEdit" name="odDataWykonaniaDateEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDateEdit" name="doDataWykonaniaDateEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
